staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/helpers.go

```diff
@@ -256,11 +272,11 @@ func extractOwnerReference(v map[string]interface{}) metav1.OwnerReference {
 	// though this field is a *bool, but when decoded from JSON, it's
 	// unmarshalled as bool.
 	var controllerPtr *bool
-	if controller, ok := NestedBool(v, "controller"); ok {
+	if controller, found, err := NestedBool(v, "controller"); found && err == nil {
```

Comment:
`err == nil && found` would be more logical.

Reply:
Done.